variables.tf

```diff
@@ -184,6 +151,32 @@ variable "capacity_type" {
   }
 }
 
+variable "block_device_mappings" {
+  type        = list(any)
+  description = <<-EOT
+    List of block device mappings for the launch template.
+    Each list element is an object with a `device_name` key and any keys
```

Comment:
```suggestion
    Each list element is an object with a `device_name` key and any other keys
```

Reply:
`device_name` is not a key in the `ebs` block.